tools/firmware-utils/patches/001-add-sifiveu-guid-types.patch

```diff
@@ -0,0 +1,38 @@
+diff -ruN firmware-utils-2022-02-28-002cfaf0.old/src/ptgen.c firmware-utils-2022-02-28-002cfaf0/src/ptgen.c
+--- firmware-utils-2022-02-28-002cfaf0.old/src/ptgen.c	2022-04-23 19:02:07.307896842 +0200
++++ firmware-utils-2022-02-28-002cfaf0/src/ptgen.c	2022-04-22 18:48:54.477970950 +0200
```

Comment:
Please add the code directly to the firmwae-utils project here too:
https://git.openwrt.org/?p=project/firmware-utils.git;a=summary

It is fine if you add it now as a patch, but when someone updates the firmwae-utils  project this should got away.